doc/controls/AutoLayoutControl.md

```diff
@@ -44,3 +44,4 @@ PrimaryAlignment|AutoLayoutPrimaryAlignment|Either Auto, the child grid take onl
 CounterAlignment|AutoLayoutAlignment|Indicates where an element should be displayed on the counter axis of the parent orientation relative to the allocated layout slot of the parent element
 PrimaryLength|double| Set the grid size in parent orientation [Deprecated should Height and Width instead] 
 CounterLength|double|Sets the child size in parent counter orientation [Deprecated should Height and Width instead] 
+IsIndependantLayout | bool | Make the child independant of the AutoLayout positionning. Should not be use other attached Properties. Reflet the Absolute Position option from Figma
```

Comment:
"Should not be used with other Attached Properties"

Meaning this won't work if we have the PrimaryAlignment/CounterAlignment/PrimaryLength/CounterLength properties attached? Would this property take precedence? Should we force the AutoLayout to ignore all of those other attached properties if this one is set? Unless that's what it's doing and I didn't notice :)

Reply:
That a good question and I took a second look to make sure :) 
This property take precedence, the other attached properties will be used.